jetty-embedded-12-ee10/src/main/java/org/jboss/arquillian/container/jetty/embedded_12_ee10/JettyEmbeddedContainer.java

```diff
@@ -253,6 +255,13 @@ public ProtocolMetaData deploy(final Archive<?> archive) throws DeploymentExcept
             deployer.removeApp(app);
             WebAppContext webAppContext = getWebAppContext(app);
 
+            // Jetty setup telling Jetty's CdiDecoratingListener how to operate.
+            webAppContext.setInitParameter(CdiServletContainerInitializer.CDI_INTEGRATION_ATTRIBUTE, CdiDecoratingListener.MODE);
+            // jetty setup for layer between Weld and Jetty.
+            webAppContext.addServletContainerInitializer(new CdiServletContainerInitializer());
+            // Weld's org.jboss.weld.environment.servlet.EnhancedListener should be discovered automatically
+            // and will start up Weld container so long as there is an archive with bean in it
```

Comment:
This is only true if the jetty-ee10-annotations jar is present in this runtime.

Reply:
Good point.
I can add it explicitly but how is Jetty going to react if its both, added here and then discovered?

Reply:
I've added it explicitly and adjusted the comment accordingly